Testing/SystemTests/tests/framework/SNSPowderRedux.py

```diff
@@ -85,7 +85,7 @@ def runTest(self):
             CalibrationFile=self.cal_file,
             CharacterizationRunsFile=self.char_file,
             LowResRef=15000,
-            RemovePromptPulseWidth=50,
```

Comment:
Please undo the changes to the test inputs. The entirety of `SNSPowderRedux.py` needs to go back to the way it was